foo() ->
  try a
  catch<caret>